### ISSUE TITLE
[PATCH] native_midi_macosx: support SDL_SOUNDFONTS

### ISSUE THREAD

**Issue by SDLBugzilla** [libsdl-org/SDL_native_midi#15]
# This bug report was migrated from our old Bugzilla tracker.

These attachments are available in the static archive:

* [native_midi_macosx: support SDL_SOUNDFONTS (native_midi_macosx_set_soundfont.patch, text/plain, 2020-11-24 22:07:14 +0000, 3917 bytes)](https://bugzilla.libsdl.org/attachment.cgi?id=4542)

**Reported in version:** unspecified
**Reported for operating system, platform:** Mac OS X (All), All

# Comments on the original bug report:

On 2020-11-24 22:07:14 +0000,  wrote:

> Created attachment 4542
> native_midi_macosx: support SDL_SOUNDFONTS
> 
> Adds support for setting soundfont via SDL_SOUNDFONTS and friends in native_midi_macosx backend.

On 2020-12-01 21:49:22 +0000, Sam Lantinga wrote:

> Fixed, thanks!
> https://hg.libsdl.org/SDL_mixer/rev/3189c67fb0d9

**Comment by icculus** [comment]
> I agree this is scary, but I'm not sure what can be done. @icculus, any ideas?

Yes, we can transfer this to SDL_native_midi and never think about it again.  :P
